packages/components/pie-toggle-switch/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@justeattakeaway/pie-toggle-switch",
+  "description": "PIE Design System Toggle Switch built using Web Components",
+  "version": "0.0.0",
+  "type": "module",
+  "main": "dist/index.js",
+  "module": "dist/index.js",
+  "types": "dist/types/index.d.ts",
+  "scripts": {
+    "build": "yarn build:wrapper pie-toggle-switch && run -T vite build",
+    "lint:scripts": "run -T eslint .",
+    "lint:scripts:fix": "yarn lint:scripts --fix",
+    "lint:style": "run -T stylelint ./src/**/*.{css,scss}",
+    "lint:style:fix": "yarn lint:style --fix",
+    "watch": "run -T vite build --watch",
+    "test": "echo \"Error: no test specified\" && exit 0",
+    "test:ci": "yarn test"
+  },
+  "author": "JustEatTakeaway.com - Design System Web Team",
+  "license": "Apache-2.0",
+  "devDependencies": {
+    "@justeattakeaway/pie-components-config": "workspace:*"
+  },
+  "dependencies": {
+    "@justeattakeaway/pie-webc-core": "workspace:*"
+  },
+  "volta": {
+    "extends": "../../../package.json"
+  }
```

Comment:
I think we need to add the browserslist config to our web components by the looks of it? Can't see it being referenced here.

Could you make a ticket to add this if so?